tests/handlers/test_directory.py

```diff
@@ -505,10 +522,24 @@ def test_allowed_with_publication_permission(self):
             self.allowed_user_id,
             tok=self.allowed_access_token,
             extra_content=self.data,
-            is_public=False,
+            is_public=True,
             expect_code=200,
         )
 
+    def test_denied_publication_with_invalid_alias(self):
+        """
+        Try to create a room, register an alias for it, and publish it,
+        as a user WITH permission to publish rooms.
+        (This is used as both a standalone test & as a helper function.)
```

Comment:
Is it?

Reply:
Arg, I copied and pasted this, removed it, deleted it all, then redid it. :(